src/components/Tabs.react.js

```diff
@@ -287,17 +293,22 @@ export default class Tabs extends Component {
                         :global(.tab-container--vert .tab) {
                             width: auto;
                             border-right: none !important;
-                            border-bottom: none;
+                            border-bottom: none !important;
```

Comment:
Is this overriding a `border` property set inline with JS? If not the `!important` could probably be avoided